bool cmd
geometry_msgs/Pose targetPose
---
bool result